src/ip/Address.h

```diff
@@ -34,12 +35,41 @@
 namespace Ip
 {
 
+class Address;
+
+class AddressText
+{
+public:
+    explicit AddressText(const Address &ip, bool printPort = false, bool printBrackets = true);
+    AddressText &withPort(bool b = false)
+    {
+        printPort_ = b;
+        return *this;
+    };
+    AddressText &bracketed(bool b = true)
```

Comment:
Please document this and other methods.

In this method description, please disclose that bracketed(true) does not imply the address will be printed with brackets. This behavior is rather counter-intuitive!

In this method description, please disclose that brackets (if any) are going to surround only the IP porting of address.

Reply:
Making it explicit that brackets only apply to IPv6 addresses